src/main/java/com/capitalone/dashboard/service/CmdbServiceImpl.java

```diff
@@ -27,10 +27,10 @@ public CmdbServiceImpl(CmdbRepository cmdbRepository) {
     public Page<Cmdb> configurationItemsByTypeWithFilter(String itemType, String filter, Pageable pageable) {
 
         Page<Cmdb> configItemString;
-
         if( StringUtils.isNotEmpty( filter ) ){
 
-            List<Cmdb> cmdbList = cmdbRepository.findAllByConfigurationItemContainingOrCommonNameContainingAllIgnoreCase(filter,filter);
+            List<Cmdb> cmdbList = cmdbRepository.findAllByConfigurationItemContainingOrConfigurationKeyContainingOrLegacyServiceManagerNameContainingOrCommonNameContainingAllIgnoreCase
```

Comment:
can you check the performance of this call? I doubt it might make it slow

Reply:
performance is tested, added indexes in cmdb.